csrc/python_frontend/python_bindings.cpp

```diff
@@ -1954,6 +1994,48 @@ void initNvFuserPythonBindings(PyObject* module) {
   NVFUSER_PYTHON_BINDING_CAST_OP("cast", castOp)
 #undef NVFUSER_PYTHON_BINDING_CAST_OP
 
+#define NVFUSER_PYTHON_BINDING_RANDOM_DIST_OP( \
```

Comment:
nitpick: maybe a nested macro to avoid boiler plate.

something like
```
#define NVFUSER_ALL_VECTOR_TYPES(fn, ...) \
  fn(Vector, __VA_ARGS__); \
  fn(py::list, __VA_ARGS__); \
  fn(py::tuple, __VA_ARGS__);

#define NVFUSER_RANDOM_DIST_OP_HELPER(vec_type, op_str, op_type, arg1_str, arg2_str)       \
  nvf_ops.def(                                 \
      op_str,                                  \
      random_dist_op_fn<vec_type, op_type>,      \
      py::arg(arg1_str),                       \
      py::arg(arg2_str),                       \
      py::arg("shape"),                        \
      py::kw_only(),                           \
      py::arg("rng_seed") = py::none(),        \
      py::arg("rng_offset") = py::none(),      \
      py::arg("dtype") = DataType::Float,      \
      py::return_value_policy::reference);

#define NVFUSER_PYTHON_BINDING_RANDOM_DIST_OP(...)       \
  NVFUSER_ALL_VECTOR_TYPES(NVFUSER_RANDOM_DIST_OP_HELPER, __VA_ARGS__)
```

Reply:
Clever!